apps/frontend/src/components/TodayMeetings.tsx

```diff
@@ -55,7 +60,7 @@ export const TodayMeetings: React.FC<TodayAgendaProps> = ({ selectedDate }) => {
           <React.Fragment key={index}>
             <li className="text-[#DCDCDD]/80 text-lg font-medium">{item.title}</li>
             <p>{item.time}, {item.duration} min</p>
-            <a href={item.link} className="text-[#DCDCDD] mt-4 mb-8 flex justify-start items-center gap-2">
+            <a href={item.link} target="_blank" className="text-[#DCDCDD] mt-4 mb-8 flex justify-start items-center gap-2">
```

Comment:
Can you use <Link></Link> from next.js here.

Reply:
![Screenshot From 2024-10-03 16-49-00](https://github.com/user-attachments/assets/7b237fb6-26b5-49df-b201-4ba776369b0d)
{item.link} is external link that's why i choosed anchor over Link im not doing any client side routing. @madhav-relish